app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
Actually scratch what I said earlier. You are 100% right. I had to reseed the database because enumerize actually mutates the Patient and User fields. I'm going to flag an extended issue on this.